src/auth/screens/auth-profile.screen.js

```diff
@@ -60,90 +60,105 @@ class AuthProfile extends Component {
-                title="Email"
-                titleStyle={styles.listTitle}
-                leftIcon={{
-                  name: 'mail',
-                  color: colors.grey,
-                  type: 'octicon',
-                }}
-                subtitle={user.email}
-                subtitleStyle={styles.listSubTitle}
-                onPress={() =>
-                  Communications.email([user.email], null, null, 'Hi!', '')}
-                underlayColor={colors.greyLight}
-              />
-            </SectionList>
-
-            <SectionList
-              title="WEBSITE"
-              noItems={!user.blog || user.blog === ''}
-              noItemsMessage={'No website associated with account'}
-            >
-              <ListItem
-                title="Website"
-                titleStyle={styles.listTitle}
-                leftIcon={{
-                  name: 'link',
-                  color: colors.grey,
-                  type: 'octicon',
-                }}
-                subtitle={user.blog}
-                subtitleStyle={styles.listSubTitle}
-                onPress={() => Communications.web(this.getUserBlog(user.blog))}
-                underlayColor={colors.greyLight}
-              />
-            </SectionList>
-
-            <SectionList
-              title="ORGANIZATIONS"
-              noItems={orgs.length === 0}
-              noItemsMessage={'No organizations'}
-            >
-              {orgs.map(item =>
-                <UserListItem
-                  key={item.id}
-                  user={item}
-                  navigation={navigation}
+              <SectionList
+                title="WEBSITE"
+                noItems={!user.blog || user.blog === ''}
+                noItemsMessage={'No website associated with account'}
+              >
+                <ListItem
+                  title="Website"
+                  titleStyle={styles.listTitle}
+                  leftIcon={{
+                    name: 'link',
+                    color: colors.grey,
+                    type: 'octicon',
+                  }}
+                  subtitle={user.blog}
+                  subtitleStyle={styles.listSubTitle}
+                  onPress={() =>
+                    Communications.web(this.getUserBlog(user.blog))}
+                  underlayColor={colors.greyLight}
                 />
-              )}
-            </SectionList>
-          </ParallaxScroll>}
+              </SectionList>
+
+              <SectionList
+                title="ORGANIZATIONS"
+                noItems={orgs.length === 0}
+                noItemsMessage={'No organizations'}
+              >
+                {orgs.map(item =>
+                  <UserListItem
+                    key={item.id}
+                    user={item}
+                    navigation={navigation}
+                  />
+                )}
+              </SectionList>
+            </ParallaxScroll>}
+        </ScrollView>
       </ViewContainer>
     );
   }
```

Comment:
This has got quite a few conflicts and is using a pretty old version of the file. For example, most of these SectionList's can be replaced with the `<EntityInfo />` component.